test/helpers/test_helpers.dart

```diff
@@ -1,8 +1,6 @@
 // ignore_for_file: talawa_api_doc
 // ignore_for_file: talawa_good_doc_comments
 
-// ignore_for_file: deprecated_member_use
```

Comment:
Why was this removed? @literalEval please comment

Reply:
Because this is the deprecation that @ALIRAZA17 is trying to fix.

Reply:
Yes this is the deprecated version we are using and I have updated it to the new one